### PR TITLE
Reimpl `MagickWand::set_image_pixels`

### PR DESCRIPTION
**Pull request by NobodyXu**
Signed-off-by: Jiahao XU <Jiahao_XU@outlook.com>

### CODE REVIEW

File: graphicsmagick/src/wand/magick.rs [hunk 1]
```diff
@@ -4584,29 +4670,47 @@ mod tests {
...
-
         assert!(!pixels.is_empty());
+
+        // Round trip tests
+        for map in ["RGB", "RGBA"] {
+            let pixels = mw.get_image_pixels::<T>(0, 0, columns, rows, map).unwrap();
+
+            let import = MagickWandImportSlice::new(columns, rows, map, &pixels).unwrap();
+
+            let mut wand = new_logo_magick_wand();
```
Comment (NobodyXu) [inline graphicsmagick/src/wand/magick.rs:4695]: This is probably not the best way to do it.
I don't know how to create a new `wand` containing an empty image.

Comment (NobodyXu) [reply graphicsmagick/src/wand/magick.rs:4695]: @jmjoy Do you know how to create an empty image suitable for importing?
It seems that just calling `MagickWand::new` is not enough for this.

Comment (jmjoy) [reply graphicsmagick/src/wand/magick.rs:4695]: Use [DrawingWand::rectangle](https://docs.rs/graphicsmagick/latest/graphicsmagick/wand/drawing/struct.DrawingWand.html#method.rectangle)?

Comment (NobodyXu) [reply graphicsmagick/src/wand/magick.rs:4695]: Hmmm, I will have a try.

Comment (NobodyXu) [reply graphicsmagick/src/wand/magick.rs:4695]: @jmjoy That didn't work.

Comment (jmjoy) [reply graphicsmagick/src/wand/magick.rs:4695]: :cry:

**Review by jmjoy — APPROVED** [review]

**Comment by jmjoy** [comment]
Feel free to merge.

**Comment by NobodyXu** [comment]
> Feel free to merge.

Hmmm, I don't have the permission to merge though.

**Comment by jmjoy** [comment]
> > Feel free to merge.
> 
> Hmmm, I don't have the permission to merge though.

Permission added.

**Comment by NobodyXu** [comment]
@jmjoy Thanks!

**Pull request merged**